### DISCUSSION TITLE
Image loop

### DISCUSSION THREAD

**Discussion by loidnoir** [maximmax42/Discord-CustomRP#137] [Ideas]
Add multiple images with some interval and loop through them

**Comment by maximmax42** [comment]
On my todo list, not sure how to implement it interface-wise.

**Reply by loidnoir** [reply]
yeah I was thinking about it too, you can make like list with inputs with, like up to 5 image input

**Reply by loidnoir** [reply]
BTW wanna let you I'm working on Armenian language translation, almost ready to contribute :D

**Reply by maximmax42** [reply]
Are you doing it manually, instead of translating on POEditor?

**Reply by loidnoir** [reply]
yeah, doing manually

**Reply by maximmax42** [reply]
Well uh, I would prefer if you did it via POEditor instead. But I guess I can make it work like that too.

**Reply by loidnoir** [reply]
no problem, I'll use POEditor than, also I noticed you're from Russia, very cool. I'd prefer to talk Russian but I think it's better to continue in English here
